docs/src/cisos/rbac.md

```diff
@@ -1 +1,21 @@
 # RBAC
+
+The [metal-api](https://github.com/metal-stack/metal-api) offers three different user roles for authorization:
+
+- `Admin`
+- `Edit`
+- `View`
+
+As part of [MEP-4](../developers/proposals/MEP4/README.md), significant work is underway to introduce more fine-grained access control mechanisms within metal-stack, enhancing the precision and flexibility of permission management.
+
+To ensure that internal components interact securely with the metal-api, metal-stack assigns specific roles to each service based on the principle of least privilege.
+
+| Component              | Role  |
+| ---------------------- | ----- |
+| image-cache            | View  |
+| Gardener               | Admin |
+| metal-bmc              | Edit  |
+| metal-core             | Edit  |
+| metal-hammer           | View  |
+| metal-metrics-exporter | Admin |
+| pixiecore              | View  |
```

Comment:
- [metal-ccm](https://github.com/metal-stack/metal-ccm)
- [metal-image-cache-sync](https://github.com/metal-stack/metal-image-cache-sync)
- [machine-controller-manager-provider-metal](https://github.com/metal-stack/machine-controller-manager-provider-metal)
- [gardener-extension-provider-metal](https://github.com/metal-stack/gardener-extension-provider-metal)
- [firewall-controller-manager](https://github.com/metal-stack/firewall-controller-manager)
- [pixie](https://github.com/metal-stack/pixie)
- [cluster-api-provider-metal-stack](https://github.com/metal-stack/cluster-api-provider-metal-stack) (at least edit)
- [metal-console](https://github.com/metal-stack/metal-console)

[metalctl](https://github.com/metal-stack/metalctl) is not a microservice and has a flexible role.